OpenID_Connect/set_oidc_config.sh

```diff
@@ -1,6 +1,23 @@
 #!/bin/bash
 DEFAULT_SDC_API_URL='https://app.sysdigcloud.com'
 
+SYSDIG_PRODUCT=""
+PRODUCT_URL=""
+while [ "z${SYSDIG_PRODUCT}" = "z" ] ; do
+  echo -n "Specify Product (monitor or secure) (required): "
+  read SYSDIG_PRODUCT
+  while [[ "$SYSDIG_PRODUCT" != "monitor" && "$SYSDIG_PRODUCT" != "secure" ]]
+   do
+     echo -n "Specify Product (monitor or secure) (required): "
+     read SYSDIG_PRODUCT
+   done
+  if [ "$SYSDIG_PRODUCT" == 'monitor' ];then
+      PRODUCT_URL='openid'
+  elif [ "$SYSDIG_PRODUCT" == secure ]; then
+    PRODUCT_URL='openid/?product=SDS'
```

Comment:
please indent consistently

Reply:
@mbreitung: Per my previous comment, this PR should probably just be canceled. The support for Secure is already in the current master version of the scripts.